python/letsql/tests/test_ml.py

```diff
@@ -298,3 +298,20 @@ def test_make_quickgrove_udf__repr(mixed_model_path):
 
     for info in expected_info:
         assert info in repr_str
+
+
+def test_quickgrove_hyphen_name():
+    from letsql.expr.relations import into_backend
+
+    model_name = "diamonds-model"
+    model_path = ls.options.pins.get_path(model_name)
```

Comment:
should we be using pins to test this? perhaps, its cleaner to use `mixed_model_path` fixture and then renaming or create a new fixture with renaming.

Reply:
Yes, using the `mixed_model_path` is better